service/raft/raft_group0_client.cc

```diff
@@ -372,4 +373,44 @@ db::system_keyspace& raft_group0_client::sys_ks() {
     return _sys_ks;
 }
 
+raft_group0_client::query_result_guard::query_result_guard(utils::UUID query_id, raft_group0_client& client)
+    : _query_id{query_id}, _client{&client} {
+    auto it = _client->_results.emplace(_query_id, std::nullopt);
+    if (!it.second) {
+        on_internal_error(logger, "query_result_guard::query_result_guard: creating an entry for the result doesn't succeed");
```

Comment:
"doesn't succeed" is a weird error.
Provide the specifics: there was an unexpected entry there
(how could there be an entry if we generate a fresh UUID?)

Reply:
Changed to `"query_result_guard::query_result_guard: there is another query_result_guard alive with the same query_id"`